pkg/kubectl/cmd/create_rolebinding_test.go

```diff
@@ -49,12 +49,12 @@ func TestCreateRoleBinding(t *testing.T) {
 		Subjects: []rbac.Subject{
 			{
 				Kind:     rbac.UserKind,
-				APIGroup: "rbac.authorization.k8s.io",
+				APIGroup: rbac.GroupName,
```

Comment:
revert changes in _test.go files.... we want to ensure the currently accepted values continue to work